src/web3/evm/call.ts

```diff
@@ -387,6 +397,9 @@ export async function callSmartContract(
                 transactionHash: "0x1122334455667788990011223344556677889900112233445566778899001122",
               }),
           contractAddress: input.fields.contractAddress,
+          _grinderyAccounting: BigNumber.from(ACCOUNTING_SIMPLE_ACTIONS)
+            .mul(BigNumber.from(CHAIN_MAPPING_ACCOUNTING[input.fields.chain]))
```

Comment:
Maybe use a default value so that we don't have to list every chain in the table? Something like:
```suggestion
            .mul(BigNumber.from(CHAIN_MAPPING_ACCOUNTING[input.fields.chain] || DEFAULT_TX_COST_RATE))
```

Reply:
@SAPikachu Done, I've just added a default value that is set up as 1 (so that no specific coefficient is applied).